python/cuml/linear_model/elastic_net.pyx

```diff
@@ -219,13 +219,13 @@ class ElasticNet(Base, RegressorMixin):
                                        'type': 'dense',
                                        'description': 'Predicted values',
                                        'shape': '(n_samples, 1)'})
-    def predict(self, X, convert_dtype=False):
+    def predict(self, X, convert_dtype=True):
```

Comment:
I wonder if in general adding a pytest to test the default value around might be a better strategy to test this? I.e. a test that goes around all the predicts, if it finds convert_dtype check that the default is True. What do you think? It could be a future improvement for sure.

Reply:
This is a good idea. I think we'd also want to go beyond testing the default value though, as the value in some cases wasn't being passed through so that test would have "silently failed". Initially, I was going to go the route of the `fit_function` tests and use `class enumerator` object. But I thought it might get a bit messy having to selectively filter out various estimators. 

I'd vote for your suggestion that we clean this up in the follow up (I plan to do this for SVC(probability=True) 👍

Reply:
Resolving; going to include in a follow up.